aea/cli/generate.py

```diff
@@ -45,44 +50,62 @@ def generate(
 
 @generate.command()
 @click.argument("protocol_specification_path", type=str, required=True)
+@click.option(
+    "--l",
+    "language",
+    type=str,
```

Comment:
```suggestion
    type=click.Choice(SUPPORTED_PROTOCOL_LANGUAGES)
```

where `SUPPORTED_PROTOCOL_LANGUAGES` is a list defined in constants.

Reply:
Done